cmd/csi-node-driver-registrar/main.go

```diff
@@ -101,14 +92,6 @@ func newRegistrationServer(driverName string, endpoint string, versions []string
 func (e registrationServer) GetInfo(ctx context.Context, req *registerapi.InfoRequest) (*registerapi.PluginInfo, error) {
 	klog.Infof("Received GetInfo call: %+v", req)
 
-	// on successful registration, create the registration probe file
-	err := util.TouchFile(registrationProbePath)
```

Comment:
There's test code for this function that we can remove

Reply:
Oops I didn't even think about tests, thx.